wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/shuffleboard/Robot.java

```diff
@@ -0,0 +1,62 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.examples.shuffleboard;
+
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.IterativeRobot;
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+public class Robot extends IterativeRobot {
+  private final DifferentialDrive m_tankDrive = new DifferentialDrive(new Spark(0), new Spark(1));
+  private final Encoder m_leftEncoder = new Encoder(0, 1);
+  private final Encoder m_rightEncoder = new Encoder(2, 3);
+
+  private final Spark m_elevatorMotor = new Spark(2);
+  private final AnalogPotentiometer m_elevatorPot = new AnalogPotentiometer(0);
+  private NetworkTableEntry m_maxSpeed;
+
+  @Override
+  public void robotInit() {
+    // Add a 'max speed' widget to a tab named 'Configuration', using a number slider
+    // The widget will be placed in the second column and row and will be two columns wide
+    m_maxSpeed = Shuffleboard.getTab("Configuration")
+                           .add("Max Speed", 1)
+                           .withWidget("Number Slider")
+                           .withPosition(1, 1)
```

Comment:
Note: add `withProperties(Map.of("min", 0, "max", 1))` when we move to Java 10/11